code/modules/antagonists/blob/blob/blobstrains/_blobstrain.dm

```diff
@@ -0,0 +1,61 @@
+GLOBAL_LIST_INIT(valid_blobstrains, subtypesof(/datum/blobstrain) - list(/datum/blobstrain/reagent, /datum/blobstrain/multiplex))
+
+/datum/blobstrain
+	var/name
+	var/description
+	var/color = "#000000"
+	var/complementary_color = "#000000" //a color that's complementary to the normal blob color
+	var/shortdesc = null //just damage and on_mob effects, doesn't include special, blob-tile only effects
+	var/effectdesc = null //any long, blob-tile specific effects
+	var/analyzerdescdamage = "Unknown. Report this bug to a coder, or just adminhelp."
+	var/analyzerdesceffect = "N/A"
+	var/blobbernaut_message = "slams" //blobbernaut attack verb
+	var/message = "The blob strikes you" //message sent to any mob hit by the blob
+	var/message_living = null //extension to first mob sent to only living mobs i.e. silicons have no skin to be burnt
+	var/core_regen = 2
+	var/resource_delay = 0
+	var/point_rate = 2
+	var/mob/camera/blob/overmind
+
+/datum/blobstrain/New(mob/camera/blob/new_overmind)
+	if (!istype(new_overmind))
+		stack_trace("blobstrain created without overmind")
```

Comment:
`QDEL_IN(30)` so that it obviously doesn't work perhaps?

Reply:
this might happen as a result of an admin doing admin things, we still want to find out if it happens but it's not absolutely always an unrecoverable state